NEWS.md

```diff
@@ -1,4 +1,12 @@
-## 2.10.0 - Unreleased
+## 3.1.0 - Unreleased
+
+## 3.0.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was to update vertx version
+
+[Full Changelog](https://github.com/folio-org/edge-orders/compare/v2.9.0...v3.0.0)
+
+### Dependencies
+* Bump `vertx` from `4.3.4` to `4.5.4`
```

Comment:
lets add here also new version of edge-common and update description mentioning edge-common

Reply:
added